docs/extended-features/server-side-actions.md

```diff
@@ -74,182 +72,173 @@ func (c *ComponentExample) Actions() kyoto.ActionMap {
+  <button onclick="{{ action 'ExampleAction' }}">Click Me</button>
 </div>
 {{ end }}
 ```
 
-#### On load trigger
+#### On Load Trigger
 
-Library provides a way to trigger an action on page load. May be useful for components lazy loading. This feature is implemented as `ssa:onload` HTML attribute and accepts just an action name.
+The library provides a way to trigger an action on page load. This may be useful for components' lazy loading. This feature is implemented as `ssa:onload` HTML attribute and accepts just an action name.
 
-Usage:  
+Usage:
 
 ```html
 {{ define "ComponentExample" }}
-<div {{ componentattrs . }} ssa:onload="Load">
-    ...
-</div>
+<div {{ componentattrs . }} ssa:onload="Load">...</div>
 {{ end }}
 ```
 
-#### Intersection trigger
+#### Intersection Trigger
 
-You can use `ssa:onintersect` HTML attribute to trigger an action on element intersection.
-This functionality was built on top of the browser's built-in `IntersectionObserver`.  
+You can use the `ssa:onintersect` HTML attribute to trigger an action on element intersection.
+This functionality was built on top of the browser's built-in `IntersectionObserver`.
 
 Usage:
 
 ```html
 {{ define "ComponentExample" }}
-<div {{ componentattrs . }} ssa:onintersect="OnIntersect">
-    ...
-</div>
+<div {{ componentattrs . }} ssa:onintersect="OnIntersect">...</div>
 {{ end }}
 ```
 
-#### Poll trigger
+#### Poll Trigger
 
-For components that must to be updated over the time (f.e. charts, stats, etc), library have polling mechanism, based on browser's built-in `setInternal`.  
+For components that must to be updated over time (f.e. charts, stats, etc), the library has a polling mechanism, based on browser's built-in `setInternal`.  
 You can use this trigger with `ssa:poll` and `ssa:poll.interval` HTML attributes.
 
 Usage:
 
 ```html
 {{ define "ComponentExample" }}
-<div {{ componentattrs . }} ssa:poll="Poll" ssa:poll.interval=1000>
-    ...
-</div>
+<div {{ componentattrs . }} ssa:poll="Poll" ssa:poll.interval="1000">...</div>
 {{ end }}
 ```
 
-#### Control display on action call
+#### Control Display on Action Call
 
 Because `kyoto` makes a roundtrip to the server every time an action is triggered on the page,
-there are cases when the page may not react immediately to a user event (like a click).
-That's why library provides a way to easily control display attribute on action call.
+there are cases where the page may not react immediately to a user event (like a click).
+That's why the library provides a way to easily control display attributes on action call.
 You can use `ssa:oncall.display` HTML attribute to control display during action call.
-In the end of an action, layout will be restored.  
+At the end of an action the layout will be restored.
 
 !!! note
-    Don't forget to set default display for loading elements like spinners and loaders
+Don't forget to set default display for loading elements like spinners and loaders
```

Comment:
This spacing created as part of the `mkdocs` formatting.

![image](https://user-images.githubusercontent.com/17050536/143931881-e6ba46e2-c08e-435d-93fa-944e248fc9b8.jpeg)